### PR TITLE
Document behavior of the MicroVG FreeType LL implementation

### PR DESCRIPTION
**Pull request by rlouedec-microej**

### CODE REVIEW

File: VEEPortingGuide/vgCco.rst [hunk 1]
```diff
@@ -108,18 +108,51 @@ This file (a header file with some C defines) enables (or disables) and configur
...
 .. _com.microej.clibrary.thirdparty#freetype: https://forge.microej.com/artifactory/microej-developer-repository-release/com/microej/clibrary/thirdparty/freetype/
 
 This C module provides a fork of Freetype 2.11.0.
 
+Memory Heap Configuration
+-------------------------
+
 The Freetype library requires a memory Heap for Freetype internal objects allocated when a font file is loaded (see https://freetype.org/freetype2/docs/design/design-4.html). 
 The size of this heap depends on the number of fonts loaded in parallel and on the fonts themselves. 
 This size is defined by ``VG_FEATURE_FREETYPE_HEAP_SIZE_HEAP`` in ``microvg_configuration.h``.
```
Comment (emoreac-microej) [inline VEEPortingGuide/vgCco.rst:125]: Please fix `VG_FEATURE_FREETYPE_HEAP_SIZE_HEAP` by renaming it into `VG_FEATURE_FREETYPE_HEAP_SIZE`

File: VEEPortingGuide/vgCco.rst [hunk 2]
```diff
@@ -108,18 +108,51 @@ This file (a header file with some C defines) enables (or disables) and configur
...
 
 All fonts do not require the same heap size. The ``MICROVG_MONITOR_HEAP`` define in ``microvg_helper.h`` and ``MEJ_LOG_MICROVG`` and ``MEJ_LOG_INFO_LEVEL`` defines in ``mej_log.h`` can be used to monitor the Freetype heap evolution.
```
Comment (emoreac-microej) [inline VEEPortingGuide/vgCco.rst:127]: The ``MICROVG_MONITOR_HEAP`` define in ``microvg_helper.h`` and ``MEJ_LOG_MICROVG`` and ``MEJ_LOG_INFO_LEVEL`` defines in ``mej_log.h`` can be used to monitor the Freetype heap evolution.

Wording suggestion:

Freetype heap evolution can be monitored by enabling the following macros:
- ``MICROVG_MONITOR_HEAP`` defined in ``microvg_helper.h``
- ``MEJ_LOG_MICROVG`` and ``MEJ_LOG_INFO_LEVEL`` defined in ``mej_log.h``

**Review by emoreac-microej — CHANGES_REQUESTED** [review]
Only 2 feedback on this review. 
The principle you describe in 4 steps is correct.

File: VEEPortingGuide/vgCco.rst (hunk 1 above)
Comment (rlouedec-microej) [reply VEEPortingGuide/vgCco.rst:125]:
```suggestion
This size is defined by ``VG_FEATURE_FREETYPE_HEAP_SIZE`` in ``microvg_configuration.h``.
```

File: VEEPortingGuide/vgCco.rst (hunk 2 above)
Comment (rlouedec-microej) [reply VEEPortingGuide/vgCco.rst:127]: Thank you, I suggest slightly different rewording:
```suggestion
All fonts do not require the same heap size. FreeType heap usage can be monitored using the following configurations:

* ``MICROVG_MONITOR_HEAP`` defined in ``microvg_helper.h``
* ``MEJ_LOG_MICROVG`` and ``MEJ_LOG_INFO_LEVEL`` defined in ``mej_log.h``
```
WDYT ?

Comment (emoreac-microej) [reply VEEPortingGuide/vgCco.rst:127]: I like this suggestion and the other one above.
Please apply them and ping me, then I approve this PR.

**Pull request merged**